index.js

```diff
@@ -1,1144 +0,0 @@
-  "Stove"
-);
-
-let study = roomFactory(
-  "The Study",
-  "\nYou see a beautiful mahogany-lined room with a couch upon which Mr. Green and Mrs. White currently sit. You can see that the Study is connected to the Kitchen, the Billiard room and the Library.",
-  "A desk",
-  "A couch",
-  "A chair"
-);
-
-let diningRoom = roomFactory(
-  "The Dining Room",
-  "\nYou see a large Dining Room with a table and chairs, two of which contain Miss Scarlet and the cook. You can see that the Dining room is connected to the Kitchen, the Billiard room and the Lounge.",
-  "Dining Table",
-  "An empty chair"
-);
-
-let billiardRoom = roomFactory(
-  "The Billiard Room",
-  "\nYou see a rich mahogany room with a billiard table at the center. You can see that the Billiard room is connected to the Dining room and the Study.",
-  "Billiard Table",
-  "Billiard Balls",
-  "Pool Cue"
-);
-
-let lounge = roomFactory(
-  "The Lounge",
-  "\nYou see a beautifully decorated room with a bar cart, a chair and a couch upon which Colonel Mustard and Mrs. Peacock sit. You can see that the Lounge is connected to the Conservatory and the Dining room.",
-  "A couch",
-  "A chair",
-  "Bar cart"
-);
-
-let conservatory = roomFactory(
-  "The Conservatory",
-  "\nYou see a beautiful conservatory filled with lots of plants and flowers. There is an orange tree, a bamboo palm and a spider plant. There is also a large cabinet in the far corner. You can see that the Conservatory is connected to the Lounge and the Ballroom.",
-  "Orange tree",
-  "Bamboo palm",
-  "Spider plant",
-  "A large cabinet"
-);
-
-let ballroom = roomFactory(
-  "The Ballroom",
-  "\nYou see an expansive ballroom with a parquet floor and a large chandelier hanging in the center. You can see that the Ballroom is connected to the Conservatory and the Library.",
-  "Chandelier",
-  "Table",
-  "Column"
-);
-
-let library = roomFactory(
-  "The Library",
-  "\nYou see a large library with many books on the shelf. The room is dark, save for the light from the crackling fireplace. Nothing much else of interests it seems. You can see that the Library is connected to the Ballroom and the Study.",
-  "A bookshelf",
-  "A fireplace",
-  "A sofa"
-);
-
-// Item constructor ///////////////////////////////////////////////////////////////////////////////
-
-class Item {
-  constructor(name, description, takeable, action) {
-    this.name = name;
-    this.desc = description;
-    this.takeable = takeable;
-    this.action = action;
-  }
-}
-
-// Player initial inventory ////////////////////////////////////////////////////////////////////////
-
-let magnifyingGlass = new Item(
-  "A Magnifying Glass",
-  "\nAn ordinary magnifying glass used to look closer.",
-  true,
-  ""
-);
-
-// Kitchen inventory ///////////////////////////////////////////////////////////////////////////////
-
-let fryingPan = new Item(
-  "Frying Pan",
-  "\nAn ordinary frying pan. Maybe you could cook up some eggs with it.",
-  true,
-  ""
-);
```

Comment:
These are examples of some items that are created but not used because the variables of the items are not being called anywhere.

Reply:
Magnifying glass is removed because it was never used and frying pan object is now being used upon examination.